Gemfile

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+group :test do
+  gem 'mdl', '~> 0.11.0'
```

Comment:
I'd not lock the mdl version here. This is already locked in the Gemfile.lock.

Reply:
Hmm. I'd argue that this makes it more explicit that it will currently break if upgraded but on the other hand we probably want to upgrade at some point and fix the issues.

Reply:
Fixed.